packages/samples/mvp-dapp/src/app.controller.ts

```diff
@@ -112,32 +46,44 @@ router.get<never, { path: string; address: string }>('/load/:address/:path', asy
     throw new BadRequest('invalid path')
   }
 
-  const recordModel = await getRecordModel(params?.address)
+  const lock = getLock(ctx.payload.params?.address)
+  const recordModel = appRegistry.findOrBind<RecordModel>(
+    new ActorReference('record', `/${lock.codeHash}/${lock.hashType}/${lock.args}/`),
+  )
   const value = recordModel.load(`data.${params.path}`)
   if (value) {
     ctx.ok(MvpResponse.ok(MvpResponse.ok(value)))
   } else {
-    throw new MvpError('field is not found', '404')
+    throw NotFound('field is not found')
```

Comment:
The `MvpError` should be recovered, and fix the CI blame by updating the `error message template`

Reply:
https://github.com/ckb-js/kuai/blob/develop/packages/common/src/errors.ts#L105-L110 requires the `error message template`, `field is not found` here, to contain the variants specified as the second parameter